ipaserver/install/httpinstance.py

```diff
@@ -263,30 +264,36 @@ def enable_mod_nss_renegotiate(self):
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRenegotiation', 'on', False)
         installutils.set_directive(paths.HTTPD_NSS_CONF, 'NSSRequireSafeNegotiation', 'on', False)
 
-    def enable_mod_nss_ocsp(self):
+    def disable_mod_nss_ocsp(self):
+        if not sysupgrade.get_upgrade_state('httpd', 'nss_oscp_enabled'):
```

Comment:
I would reformulate the condition as `if sysupgrade.get_upgrade_state('httpd', 'nss_ocsp_enabled') is None` this correctly reflects the semantics that if the upgrade state is None, it was not touched by IPA before.

Also please move the 'nss_ocsp_enabled' literal to a top-level module constants named e.g. NSS_OCSP_ENABLED.